pkg/kubelet/cm/topologymanager/socketmask/socketmask.go

```diff
@@ -116,6 +117,17 @@ func (s *socketMask) IsEqual(mask SocketMask) bool {
 	return *s == *mask.(*socketMask)
 }
 
+//IsNarrowerThan checks if one mask is narrower than another.
```

Comment:
Nit: For those of us who aren't super experienced with bit masks (waves hand :) ), it could be useful to add one sentence to this comment expanding on what it means for one bit mask to be narrower than another.

Reply:
Sure, no problem